include/odp/api/spec/classification.h

```diff
@@ -478,6 +536,39 @@ int odp_cos_with_l3_qos(odp_pktio_t pktio_in,
 			odp_cos_t cos_table[],
 			odp_bool_t l3_preference);
 
+/**
+ * Reset statistics for a CoS queue
+ *
+ * Reset all statistics counters to zero. The queue must be assigned to the
+ * given CoS. Event queue handles can be requested with odp_cos_queue() and
```

Comment:
"Queue must be assigned to the given cos" is probably not always true right ? Same odp_queue_t can be given to multiple CoS objects.

Reply:
Removed in V3